doc/rbd/rbd-nomad.rst

```diff
@@ -73,38 +73,54 @@ Configure ceph-csi
 Setup Ceph Client Authentication
 --------------------------------
 
-Create a new user for nomad and `ceph-csi`. Execute the following and
-record the generated key::
+Create a new user for nomad and `ceph-csi`. Execute the following command and
+record the generated key:
 
-    $ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
+.. code-block:: console
+
+  $ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
     [client.nomad]
         key = AQAlh9Rgg2vrDxAARy25T7KHabs6iskSHpAEAQ==
 
 
 Configure Nomad  
 ---------------
 
-By default Nomad doesn't allow containers to use privileged mode.
-Edit the nomad configuration file by adding this configuration block to `/etc/nomad.d/nomad.hcl`::
+Configuring Nomad to Allow Containers to Use Privileged Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, Nomad doesn't allow containers to use privileged mode. We must
+configure Nomad so that it allows containers to use privileged mode. Edit the
+Nomad configuration file by adding the following configuration block to
+`/etc/nomad.d/nomad.hcl`::
 
     plugin "docker" {
         config {
         allow_privileged = true
         }
```

Comment:
```suggestion
      }
```

Reply:
Fixed.

Reply:
It didn't seem to work.  I'm looking for something like
```
plugin "docker" {
  config {
    allow_privileged = true
  }
}
```
instead of
```
plugin "docker" {
    config {
    allow_privileged = true
    }
}
```
The amount of leading indentation and the indentation width are irrelevant, what matters is proper nesting.

Reply:
I understand. This is exactly what I committed in my most recent patch.

I wonder if maybe you weren't looking at a local cache.

Let me know if it looks right to you on another attempt.

Reply:
Ah, weird.  Looks like `readthedocs.build` didn't refresh properly for some reason.  This bit looks fine but I take back my words about `prompt` followed by `code-block` seeming to work.  I'll comment in that thread.

Reply:
The prompt/code-block lack is one of the big rubs in this toolchain.

I'm testing another possible workaround (which is just the prompt directive followed by the command and then followed by a blank line and then followed by :: and then followed by the console output). We shall see what we shall see, I guess.